draft-packages/avatar/tsconfig.dist.json

```diff
@@ -1,5 +1,8 @@
 {
   "extends": "../../tsconfig.dist.json",
+  "compilerOptions": {
+    "outDir": "dist"
+  },
```

Comment:
It's weird that I need to do this, even if I put it in the root tsconfig, it won't work.